content/applications/inventory_and_mrp/manufacturing/quality_control/quality_alerts.rst

```diff
@@ -0,0 +1,74 @@
+=====================
+Create quality alerts
+=====================
+
+:doc:`Configuring quality control points <quality_control_points>` is a great way to ensure that
+quality checks are performed at routine stages during specific operations. However, quality issues
+can often appear outside of these scheduled checks. Using *Odoo Quality*, users can create quality
+alerts for issues that are not detected by automated processes.
```

Comment:
Think this doc link serves better as a `seealso`.
As well (separate), maybe we should rename the other doc's H1 to say `Add quality control points` or something like that...a little more specificity can help differentiate these two docs some more.
```suggestion
Configuring quality control points is a great way to ensure that quality checks are performed at
routine stages during specific operations. However, quality issues can often appear outside of these
scheduled checks. Using Odoo *Quality*, users can create quality alerts for issues that are not
detected by automated processes.

.. seealso::
   :doc:`Add quality control points <quality_control_points>`
```